setgithubcheck/setgithubcheck.js

```diff
@@ -10,7 +10,24 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-var config = require('./config.json');
+var fs = require('fs');
+var config_path = './config.json';
+var config;
+if (fs.existsSync(config_path)) {
+  config = require('./config.json');
+} else if (process.env.SIGN_REFRESH_TOKEN && process.env.SIGN_CLIENT_ID && process.env.SIGN_CLIENT_SECRET && process.env.GITHUB_KEY && process.env.GITHUB_APP_ID) {
+  config = {
+    signRefreshToken: process.env.SIGN_REFRESH_TOKEN,
+    signClientID: process.env.SIGN_CLIENT_ID,
+    signClientSecret: process.env.SIGN_CLIENT_SECRET,
+    githubKey: process.env.GITHUB_KEY,
+    githubAppId: process.env.GITHUB_APP_ID
+  };
+} else if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST.length) {
```

Comment:
i like this. Essentially don't add any config env variables for PRs.

Question, what happens when the config is empty? Will it error out below when trying to grab variables from it?

Reply:
Yeah, as soon as we start adding tests that dive deeper into the actions, we will hit errors. I think we can avoid it by stubbing out the actual calls to the github api, though.

Maybe I put together the stubbing and a few more tests so we have a template to follow for future tests, and also ensure we don't hit this issue?